tests/Resources/Page/HybridCloudConsole/Rhosak.robot

```diff
@@ -183,7 +183,7 @@ Clean Up RHOSAK
   Enter Stream Topics Section
   Delete Stream Topic  topic_to_delete=${topic_to_delete}
   Menu.Navigate To Page    Streams for Apache Kafka   Kafka Instances
-  Wait For HCC Splash Page
+  Run Keyword And Ignore Error    Wait For HCC Splash Page
```

Comment:
Maybe Continue on Failure would be better?

Reply:
`Continue on Failure` would make the TC fail

Reply:
Yes we would keep seeing the failure but at least we have a reminder that something wrong is going on there while still executing the TC

Reply:
well, as it is done now, it continue on failure automatically (because it is a teardown). The point is that if the splash page does not appear, it doesn't mean the TC will fail in this case, but it can success anyway (as it is happening). If the splash page keyword fails when it shouldn't, then the next steps would fail making the TC correctly failing